permissions/resource_permissions.go

```diff
@@ -336,37 +336,7 @@ func ResourcePermissions() *schema.Resource {
 		}
 		return s
 	})
-	readContext := func(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
-		id := d.Id()
-		objectACL, err := NewPermissionsAPI(ctx, m).Read(id)
-		if common.IsMissing(err) {
-			log.Printf("[INFO] %s is removed on backend", d.Id())
-			d.SetId("")
-			return nil
-		}
```

Comment:
This is missing from the new `Read` implementation. Could it regress?

Reply:
this is part of `common.Resource`, which already consistently handles it across all resources

Reply:
covered by `TestResourcePermissionsRead_NotFound`